openwisp_utils/admin_theme/templates/openwisp_utils/email_template.html

```diff
@@ -0,0 +1,112 @@
+{% load i18n %}
+{% load l10n %}
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  {% block styles %}
+  <style type="text/css">
+    body {
+      min-width: 250px;
+      background: #ffe5e5;
+      margin: 0;
+      padding: 0;
+    }
+
+    .container{
+      background: #ffe5e5;
+      height: 100%;
+      min-height: 500px;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+
+    .box{
+      width: auto;
+      padding: 60px 40px;
+      background-color: #fcfcfc;
+      box-sizing: border-box;
+      box-shadow: 0 15px 25px rgba(256,256, 256,.5);
+      border-radius: 10px;
+      margin: auto;
+    }
+
+    .logo-container{
+      text-align: center;
+      margin-bottom: 25px;
+    }
+
+    .logo {
+      max-width: 230px;
+    }
+
+    .call-to-action{
+      text-align: center;
+      margin-top: 30px;
+    }
+
+    .subject{
+      font-size: medium;
+      font-weight: 700;
+      margin: 20px 0;
+    }
+
+    .msg{
+      color: #1b1818;
+      margin: 20px 0;
+    }
+
+    .btn{
+      appearance: button;
+      text-decoration: none;
+      color: #fff!important;
+      background: #333;
+      border-radius: 4px;
+      border: 0;
+      padding: 10px 15px;
+      font-weight: 500;
+      font-size: 14px;
+      text-transform: uppercase;
+    }
+
+    .btn:hover, .btn:focus {
+      background: #000;
+      outline: 0 none;
+      background: rgba(51, 51, 51, 0.7);
+    }
+
+    a:link:hover {
+      text-decoration: none;
+    }
+  </style>
+  {% endblock styles %}
```

Comment:
```suggestion
  {% block styles %}
  {% endblock styles %}
```
If we add this here then we can override the default style instead of rewriting the same thing. Any suggestions?

Reply:
@codesankalp we define `block styles` already in line 7

Reply:
Yes, we can override with super, thanks for the clarification.